docs/core/upgrade.md

```diff
@@ -0,0 +1,130 @@
+<!--
+order: 15
+-->
+
+# In-Place Store Migrations
+
+::: warning
+Read and understand all of the in-place store migration documentation before you run a migration on a live chain.
+:::
+
+Upgrade your app modules smoothly with custom in-place migration logic. {synopsis}
+
+The Cosmos SDK uses two methods to perform upgrades. 
+
+- Exporting the entire application state to a JSON file using the `export` CLI command, making changes, and then starting a new binary with the changed JSON file as the genesis file. See the [Chain Upgrade Guide](../migrations/chain-upgrade-guide-040.md#upgrade-procedure). 
+
+- Version v0.43 and later can perform upgrades in place to significantly decrease the upgrade time for chains with a larger state. Use the [Migration Upgrade Guide](../building-modules/upgrade.md) guide to set up your application modules to take advantage of in-place upgrades.
+
```

Comment:
I remember reading a sentence saying something like "this document describes the 2nd method", imo it's still useful to add it here

Reply:
[specified method](https://github.com/cosmos/cosmos-sdk/pull/8967/files#diff-7919f886ac1c280f5bc85e12992892df7f160c4ab535e878d2f1510f6d11489cR18-R21)